src/main/java/seedu/intern/model/applicant/GraduationYearMonth.java

```diff
@@ -36,15 +36,17 @@ public GraduationYearMonth(String graduationYearMonth) {
      * Returns true if a given string is a valid grade name.
      */
     public static boolean isValidGraduationYearMonth(String test) {
-        int holder;
+        int monthHolder;
+        int yearHolder;
         if (test.matches(VALIDATION_REGEX)) {
-            holder = Integer.parseInt(test.split("/")[0]);
+            monthHolder = Integer.parseInt(test.split("/")[0]);
+            yearHolder = Integer.parseInt(test.split("/")[1]);
         } else {
             return false;
         }
 
-        // month in between 1-12 inclusive
-        return test.matches(VALIDATION_REGEX) && holder >= 1 && holder <= 12;
+        // month in between 1-12 inclusive and year lower bound of 2020
+        return test.matches(VALIDATION_REGEX) && monthHolder >= 1 && monthHolder <= 12 && yearHolder >= 2020;
```

Comment:
YearMonth have something called .now() so can just use that instead of this

Reply:
I would suggest a more lenient lower bound (maybe 1980 etc.) if a user wishes to transfer existing data into our system this might be too strict.

Reply:
Okay 1980 should be reasonable enough. Will update it.

Reply:
I feel the .now() method works better in our case or a higher lower bound should be set than 1980 as internship application is useful with a short period of usage. Either way we should just specify in UG/DG. What do you think

Reply:
A problem I found with the .now() method is that if there is even a single applicant with an invalid YearMonth and the year moves to the next year, then the application won't load the json file on launch and will launch with empty data instead. 
I'm not sure if this is an important bug to fix though since I don't know how the user can test this or if it will even happen during the PE.

So maybe the lower bound can be eg. YearMonth.now() - 5, so that there can be a bit of space in case the above happens?